drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -577,8 +577,8 @@ int nrf_wifi_if_start_zep(const struct device *dev)
 	vif_info.state = NRF_WIFI_FMAC_IF_OP_STATE_UP;
 
 	memcpy(vif_info.ifacename,
-	       dev->name,
-	       strlen(dev->name));
+	       "wlan0",
```

Comment:
why is this now hardcoded?

Reply:
RPU identifies the dev name by wlan0.

Reply:
You can change the name of the network interface by calling
```
int net_if_set_name(struct net_if *iface, const char *buf);
```
if needed. Requires that `CONFIG_NET_INTERFACE_NAME` is enabled.
By default the first wifi interface is wlan0, but that can be changed if needed.

Reply:
> RPU identifies the dev name by wlan0.

This should be taken care in the FMAC as its specific to UMAC.

Reply:
Done